js/script.js

```diff
@@ -1,70 +1,88 @@
 $(document).ready(function(){
+  //Single State Object
+  var state = {
+    currentQuestion: 0,
+    nextQuestion: function() {
+      if(this.currentQuestion >= questions.length-1) {
+        this.drinkNaming();
+        $('.question').html("It seems you would like a drink with " + this.drinkPref + " that would be a " + this.drinkName);
+        $('.responses').hide();
+        $('#startOver').show();
+      } else {
+        this.currentQuestion++;
+        askQuestion();
+      }
+    },
+    drinkPref: [],
+    drinkName: '',
+    drinkNaming: function() {
+      var word = pirateWords[Math.floor(Math.random() * pirateWords.length)];
+      var adjective = pirateAdjectives[Math.floor(Math.random() * pirateAdjectives.length)];
+      this.drinkName = adjective +'-'+ word;
+    }
+  };
 
-  //questions and possible ingredients with flavor id which is currently unused
-  var currentQuestion = -1;
+  var BartenderQuestion = function(question, ingred, flavor){
+      this.question = question;
+      this.ingredients = ingred;
+      this.flavor = flavor;
+  };
```

Comment:
Constructor function looks good. Maybe consider adding each new question you create to an existing Questions object.